atlasdb-client/src/main/java/com/palantir/atlasdb/logging/SafeLoggableDataUtils.java

```diff
@@ -54,6 +55,8 @@ static void addLoggableNamesToBuilder(
         if (IS_SAFE.test(tableMetadata.getNameLogSafety())) {
             builder.addPermittedTableReferences(ref);
         }
+        // this is a system table with empty metadata, but safe for logging.
+        builder.addPermittedTableReferences(AtlasDbConstants.DEFAULT_METADATA_TABLE);
```

Comment:
This is a little hacky but I don't see an easy alternative (and don't really fancy making `EMPTY_TABLE_METADATA` be safe), so this is fine.